### PR TITLE
feat: Update AS to latest gas schedule + decimals from sampler

### PR DESCRIPTION
**Pull request by dekz**
# Description

Updates to latest AS which returns the decimals from the sampler and has an updated gas schedule.

# Testing Instructions

[Sim](https://metabase.spaceship.0x.org/dashboard/139?run_id=deserve-warlike-ink&adjusted_amount_bought_win_tolerance__bps_=0.5)

# Checklist

-   [ ] Update [documentation](https://github.com/0xProject/website/blob/development/mdx/api/index.mdx) as needed. **Website Documentation PR:**
-   [ ] Prefix PR title with `[WIP]` if necessary.
-   [ ] Add tests to cover changes as needed.

### CODE REVIEW

**Comment by dekz** [comment]
deploy staging

File: src/services/swap_service.ts [hunk 1]
```diff
@@ -566,18 +562,17 @@ export class SwapService {
         };
     }
 
+    // tslint:disable-next-line:prefer-function-over-method
     private async _getSwapQuotePriceAsync(
```
Comment (kimpers) [inline src/services/swap_service.ts:566]: NIT: this doesn't need to be `async` anymore right?

**Review by kimpers — APPROVED** [review]
Nice! 🙏

**Comment by romainbutteaud** [comment]
not sure if you saw but the depth chart endpoint is failing. 

https://staging.api.0x.org/swap/v1/depth?buyToken=WBTC&sellToken=WETH&sellAmount=326945715602116690634799&numSamples=75&sampleDistributionBase=1.02

```
{
reason: "Invalid address: 'wbtc'"
}
```

**Comment by dekz** [comment]
> not sure if you saw but the depth chart endpoint is failing.
> 
> https://staging.api.0x.org/swap/v1/depth?buyToken=WBTC&sellToken=WETH&sellAmount=326945715602116690634799&numSamples=75&sampleDistributionBase=1.02
> 
> ```
> {
> reason: "Invalid address: 'wbtc'"
> }
> ```

Thanks this should now be fixed. Only tested with addresses 😂

**Comment by dekz** [comment]
deploy staging

**Comment by dekz** [comment]
deploy production

**Pull request merged**